qcodes/instrument_drivers/american_magnetics/AMI430.py

```diff
@@ -28,32 +30,42 @@ class AMI430(IPInstrument):
         current_ramp_limit (float): current ramp limit in ampere per second
         persistent_switch (bool): whether this magnet has a persistent switch
     """
-
     mocker_class = MockAMI430
     default_current_ramp_limit = 0.06  # [A/s]
 
     def __init__(self, name, address=None, port=None, persistent_switch=True,
-                 reset=False, current_ramp_limit=None, terminator='\r\n', testing=False, **kwargs):
+                 reset=False, current_ramp_limit=None,
+                 terminator='\r\n', testing=False, **kwargs):
+
+        print('Debug: {}, {}'.format(address, port))
 
-        if None in [address, port] and not testing:
-            raise ValueError("The port and address values need to be given if not in testing mode")
+        # if None in [address, port] and not testing:
+        #    raise ValueError("The port and address values need to be"
+        #                     " given if not in testing mode")
```

Comment:
Should this be removed?

Reply:
Yes remove this as well

Reply:
I'll tear this out as well as all references to `testing` and the old `mocker_class` since this is no longer needed.